standalone/src/models/shipping-method/zone-rate/zone-rate-draft/fields-config.ts

```diff
@@ -0,0 +1,21 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import { KeyReferenceDraft } from '@/models/commons';
+import { ShippingRateDraftRest } from '../../shipping-rate/shipping-rate-draft';
+import type { TZoneRateDraftRest, TZoneRateDraftGraphql } from '../types';
+
+const commonFieldsConfig = {
+  zone: fake(() => KeyReferenceDraft.presets.zone()),
+  shippingRates: fake(() => [ShippingRateDraftRest.random()]),
```

Comment:
This can't be populated in the common fields as each fields config should use a different nested model: `ShippingRateDraftRest | ShippingRateDraftGraphql`.